products/BellHybrid/alarms/src/actions/FrontlightAction.cpp

```diff
@@ -149,14 +161,22 @@ namespace alarms
     screen_light_control::LinearProgressModeParameters LinearProgressFrontlightAction::prepareParameters()
     {
         constexpr auto firstTargetDuration = std::chrono::seconds{5};
+
+        std::string brightnessString =
+            settings.getValue(bell::settings::PrewakeUp::brightness, settings::SettingsScope::Global);
+        float brightness              = 50.0f;
+        if (!brightnessString.empty()) {
+            brightness = frontlight_utils::fixedValToPercentage(std::stoi(brightnessString));
+        }
```

Comment:
You could consider putting this inside a function since it's repeated in `ManualFrontlightAction`.

Reply:
I just simplified this repeated code.